src/content-handlers/iiif/modules/uv-contentleftpanel-module/GalleryView.ts

```diff
@@ -64,12 +65,14 @@ export class GalleryView extends BaseView<ContentLeftPanel> {
   show(): void {
     this.isOpen = true;
     this.$element.show();
-
+  
     // todo: would be better to have no imperative methods on components and use a reactive pattern
     setTimeout(() => {
       this.galleryComponent.selectIndex(this.extension.helper.canvasIndex);
+      this.applyExtendedLabelsStyles(); 
     }, 10);
   }
+  
```

Comment:
Another unwanted blank line; there also seems to be some unnecessary trailing whitespace elsewhere in this file.